### PR TITLE
TASK: Improve documentation on Configuration and Installation

### PR DESCRIPTION
**Pull request by mad99**
Have Add a note to chapter «Installation» and change the sequence in chaper «Configuration» for better usable/find the context-part, not to skip because of note I moved this to the end and add more subtitle about context for better SERP-Result.

More Details about the reason in slack-channel «guilde-documentation»

(Second try: Sorry)

### CODE REVIEW

File: Neos.Flow/Documentation/TheDefinitiveGuide/PartII/Installation.rst [hunk 1]
```diff
@@ -222,18 +228,20 @@ machine. Add the following line to your */etc/hosts* file
...
 
-.. tip::
-	If you decided to skip setting up virtual hosts earlier on, you can
-	enable the ``Production`` context by editing the ``.htaccess`` file in the
-	``Web`` directory and remove the comment sign in front of the ``SetEnv``
-	line:
+Change Context to «Production» without Virtual Host
+---------------------------------------------------
+
+If you decided to skip setting up virtual hosts earlier on, you can enable the ``Production`` context by editing the ``.htaccess`` file in the``Web`` directory and remove the comment sign in front of the ``SetEnv`` line:
```
Comment (daniellienert) [inline Neos.Flow/Documentation/TheDefinitiveGuide/PartII/Installation.rst]: no space between the and ``Web``

File: Neos.Flow/Documentation/TheDefinitiveGuide/PartII/Configuration.rst [hunk 2]
```diff
@@ -62,6 +62,10 @@ Directory						Description
 *Configuration/Production/*		Configuration for the ``Production`` context
 ============================	==================================================
 
+.. note::
+	Setting Up Context with Virtual Host and change Context from «Development» to «Production» is explained in the next previous chapter «Installation».
```
Comment (daniellienert) [inline Neos.Flow/Documentation/TheDefinitiveGuide/PartII/Configuration.rst]: I guess previous is wrong here?

File: Neos.Flow/Documentation/TheDefinitiveGuide/PartII/Installation.rst [hunk 3]
```diff
@@ -222,18 +228,20 @@ machine. Add the following line to your */etc/hosts* file
...
 
 *.htaccess*:
 
 .. code-block:: none
 
 	# You can specify a default context by activating this option:
 	SetEnv FLOW_CONTEXT Production
+  
+.. note::
+	The concept of contexts and why do you want contexts, is explained in the	next chapter «Configuration».
```
Comment (daniellienert) [inline Neos.Flow/Documentation/TheDefinitiveGuide/PartII/Installation.rst]: I would rather write: "The concept of contexts and their benefits ..."

File: Neos.Flow/Documentation/TheDefinitiveGuide/PartII/Configuration.rst (hunk 2 above)
Comment (mad99) [inline Neos.Flow/Documentation/TheDefinitiveGuide/PartII/Configuration.rst]: Sorry, have overlooked in this (second) PR. Sequence is «Introduction, Requirements, Installation, Configuration, Modeling, ...»
This part is in the chapter Configuration, so I think «next» is wrong. It should written:

Setting Up Context with Virtual Host and change Context from «Development» to «Production» is explained in the previous chapter «Installation».

File: Neos.Flow/Documentation/TheDefinitiveGuide/PartII/Installation.rst (hunk 3 above)
Comment (mad99) [inline Neos.Flow/Documentation/TheDefinitiveGuide/PartII/Installation.rst]: +1 : better sentence

**Comment by mad99** [comment]
There is a problem with branch lower than 4.0. A 404-page was returned, when I tried a lower branch in drop-down Version-Selection.
I guess it would be good for «branch 2.3 and upwards».  Maybe anyone could move this down internal?

**Comment by albe** [comment]
I guess there are just too many changes in 4.0 which conflict when you down-target, so github blows - you need to rebase your local branch onto 3.2 and force push again, then retarget.

**Comment by mad99** [comment]
Sorry, for the triggered problems. I'm a newbie in Github-Documentation.
I write in Github-editor not on local-machine/local-branch:

Under url: https://github.com/neos/flow-development-collection/blob/master/Neos.Flow/Documentation/TheDefinitiveGuide/PartII/Configuration.rst I get an 404.html. Before in other Docu-Contribution there was no problem to change the brach and edit the Docu.

So if I change the branch, there are NO changes. Github should not blows up. But anyway, If there is no internal solution to change the branch, it will work for 4.0 and up. So it should be like this.

branch-2 3
bildschirmfoto 2017-03-22 um 17 38 29

**Comment by kdambekalns** [comment]
In the 2.3 branch `Neos.Flow` is still named `TYPO3.Flow`

**Comment by aertmann** [comment]
https://github.com/neos/flow-development-collection/blob/3.2/TYPO3.Flow/Documentation/TheDefinitiveGuide/PartII/Configuration.rst

**Comment by mad99** [comment]
Oh dear, such a simple solution... 2 seconds, if I had known/realized the relation to namespace stuff.
Then it's supposed to be: 4.0 and up.

Thanks for help: Will remember next contributing.

**Comment by albe** [comment]
Ah sorry, I misunderstood the issue it seems.

**Comment by albe** [comment]
ping - would be cool to have that in 4.1 release. If you don't have time for finishing this up @mad99 just let us know, then maybe someone else can jump on

**Comment by mad99** [comment]
@albe: Thank you for ping. 
Honestly I thought, there have to press someone from the core-team/docu-team the button to set it OK and merged.
With my: «then it supposed to be: 4.0 and up» I guess, it should be the Go-Sign from my-side.

I didn't want stress the one with «merge-button» and thought: it would be either merged some a day or deleted, because something general does not fit in the written content/concept of "re-sequencing".

So, I would have time, but I don't know what to do ...
There is the information about «This branch has conflichts that must be resolved». Pressed on «Resolve conflicts» I get the information about the conflict:
<<<<< 4.0 ...in the next previous chapter «Installation».
>>>>> 4.0 ... in the previous chapter «Installation».

I don't have possibilities to change something, also the button «mark as resolved» is grey/without functions for me.
no-possibilities-to-resolve-conflict

So I thought weeks back, it will merged from person with go-button for >= 4.0 with « ... in the previous chapter «Installation» .»
So sorry for holdup : But also – could anyone advice me, where the go-button is to find/what I have to do, to choose this version and resolve the conflict?

**Comment by mad99** [comment]
Oh dear :o((.
Tried on an other path, and works perfect.
Sorry, I thought I have to press first the button. Big idiot: only change it, and press later. So I hope it will do now!

**Comment by albe** [comment]
Great, now it just needs the adjustments that Daniel mentioned :)

**Pull request merged**